Services/Interfaces/web_interface/api/config.py

```diff
@@ -45,3 +45,16 @@ def change_reference_market_on_config_currencies():
     success, reply = models.change_reference_market_on_config_currencies(request_data["old_base_currency"],
                                                                          request_data["new_base_currency"])
     return util.get_rest_reply(flask.jsonify(reply)) if success else util.get_rest_reply(reply, 500)
+
+
+@api.api.route('/copy_trading_strategy', methods=["POST"])
```

Comment:
I would prefer something like `start_copy_trading`because here we don't really know what's the action behind this route

Reply:
right, updating it

Reply:
right, updating it